Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -505,12 +505,17 @@ instance (priority := 100) ContinuousSub.measurableSub [Sub γ] [ContinuousSub 
 instance (priority := 100) ContinuousInv.measurableInv [Inv γ] [ContinuousInv γ] :
     MeasurableInv γ := ⟨continuous_inv.measurable⟩
 
+@[to_additive]
+instance (priority := 100) ContinuousConstSMul.to_measurableConstSMul {M α} [TopologicalSpace α]
```

Comment:
The instance name is not coherent with the name of the next instance. I don't know what the naming convention requires for naming instances, but if I remember correctly often there should be an `inst` somewhere and no underscore (or no name at all!)

Reply:
The naming convention for inheritance instances (usually the ones with priority 100) is `ParentClass.toChildClass` or `ParentClass.to_childClass` depending on whether `ChildClass` is Type- or Prop-valued. cc @eric-wieser

Reply:
I agree this should start with `to`, I think we are very inconsistent about `Foo.to_bar` and `foo.toBar`, especially as I think `extends` only generates the latter. My preference would be to drop the `_` for now, and perhaps start a zulip thread about encoding this style.

Reply:
Would one of you add a few lines about this in https://leanprover-community.github.io/contribute/naming.html? In any case, this one or the next one needs fixing.